### PR TITLE
Update cross-origin isolation related articles

### PR DESCRIPTION
**Pull request by agektmr**
This update adds information about upcoming changes around COEP: credentialless and COOP: same-origin-allow-popups.

### CODE REVIEW

**Comment by agektmr** [comment]
cc: @lutzvahl

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ isolation is required if you want to use
 precision](https://developer.chrome.com/blog/cross-origin-isolated-hr-timers/),
 or the JS Self-Profiling API.
 
+If you intend to enable cross-origin isolation, evaluate the impact this will
+have on other cross-origin resources on your website, e.g. ad placements.
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:29]:
```suggestion
have on other cross-origin resources on your website, such as ad placements.
```

**Comment by lutzvahl** [comment]
LGTM

File: src/site/content/en/blog/coop-coep/index.md [hunk 2]
```diff
@@ -135,9 +143,10 @@ article](/why-coop-coep) I will provide more background and context.
 
 {% Aside %}
 This article is aimed at those who would like to get their websites ready for
-using `SharedArrayBuffer`, WebAssembly Threads, `performance.measureUserAgentSpecificMemory()`
-or the JS Self-Profiling API in a more robust manner across browser
-platforms.
+using `SharedArrayBuffer`, WebAssembly Threads,
```
Comment (mihajlija) [inline src/site/content/en/blog/coop-coep/index.md:146]:
```suggestion
using `SharedArrayBuffer`, WebAssembly threads,
```

File: src/site/content/en/blog/coop-coep/index.md [hunk 3]
```diff
@@ -167,10 +176,27 @@ enable cross-origin isolation](/cross-origin-isolation-guide/).
...
+as OAuth and payments. To mitigate this problem, we are [exploring relaxing the
+condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
+isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
+the communication with the window opened by itself will be possible.
+
+If you want to enable cross-origin isolation but are blocked by this issue, we
+recommend [registering for an origin
+trial](https://developer.chrome.com/blog/enabling-shared-array-buffer/#origin-trial)
+and wait until the new condition will be available. We are not planning to
+terminate the origin trial until this issue can be safely resolved.
```
Comment (mihajlija) [inline src/site/content/en/blog/coop-coep/index.md:197]:
```suggestion
and waiting until the new condition is available. We are not planning to
terminate the origin trial until this issue is safely resolved.
```

File: src/site/content/en/blog/coop-coep/index.md [hunk 4]
```diff
@@ -204,7 +229,7 @@ Here is what you need to do depending on the nature of the resource:
   `Cross-Origin-Resource-Policy: cross-origin` header if possible.
 * For cross origin resources that you have no control over:
     * Use the `crossorigin` attribute in the loading HTML tag if the resource is
-      served with CORS.
+      served with CORS. (ex: `<img src="***" crossorigin>`)
```
Comment (mihajlija) [inline src/site/content/en/blog/coop-coep/index.md:232]:
```suggestion
      served with CORS. (For example, `<img src="***" crossorigin>`.)
```

File: src/site/content/en/blog/coop-coep/index.md [hunk 5]
```diff
@@ -249,6 +274,25 @@ to gain access to Wasm Threads (and Shared Array Buffer) as well on Android
...
+{% Aside 'caution' %}
+
+We've been exploring ways to deploy `Cross-Origin-Resource-Policy` at scale, as
+cross-origin isolation requires all subresources to explicitly opt-in. And we
+have come up with the idea of going in the opposite direction: [a new COEP
+"credentialless" mode](https://github.com/mikewest/credentiallessness/) that
+allows loading resources without the CORP header by stripping all their
+credentials. We are working out how it should work in detail, but we hope this
+will lighten your burden of making sure the subresources are sending the
+`Cross-Origin-Resource-Policy` header.
```
Comment (mihajlija) [inline src/site/content/en/blog/coop-coep/index.md:286]:
```suggestion
credentials. We are figuring out the details of how it should work, but we hope this
will lighten your burden of making sure the subresources are sending the
`Cross-Origin-Resource-Policy` header.
```

File: src/site/content/en/blog/coop-coep/index.md [hunk 6]
```diff
@@ -249,6 +274,25 @@ to gain access to Wasm Threads (and Shared Array Buffer) as well on Android
...
+
+If you want to enable cross-origin isolation but are blocked by this, we
+recommend [registering for an origin
+trial](https://developer.chrome.com/blog/enabling-shared-array-buffer/#origin-trial)
+and wait until the new mode is available. We are not planning to terminate the
```
Comment (mihajlija) [inline src/site/content/en/blog/coop-coep/index.md:291]:
```suggestion
and waiting until the new mode is available. We are not planning to terminate the
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 7]
```diff
@@ -122,9 +130,33 @@ would be impacted.
...
 from being able to communicate with popup windows.
+
+We've been exploring ways to deploy `Cross-Origin-Resource-Policy` at scale, as
+cross-origin isolation requires all subresources to explicitly opt-in. And we
+have come up with the idea of going in the opposite direction: [a new COEP
+"credentialless" mode](https://github.com/mikewest/credentiallessness/) that
+allows loading resources without the CORP header by stripping all their
+credentials. We are working out how it should work in detail, but we hope this
+will lighten your burden of making sure the subresources are sending the
+`Cross-Origin-Resource-Policy` header.
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:145]:
```suggestion
credentials. We are figuring out the details of how it should work, but we hope this
will lighten your burden of making sure the subresources are sending the
`Cross-Origin-Resource-Policy` header.
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 8]
```diff
@@ -122,9 +130,33 @@ would be impacted.
...
+break integrations that require cross-origin window interactions such as OAuth
+and payments. To mitigate this problem, we are exploring [relaxing the
+condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
+isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
+the communication with the window opened by itself will be possible.
+
+If you want to enable cross-origin isolation but are blocked by these
+challenges, we recommend [registering for an origin
+trial](https://developer.chrome.com/blog/enabling-shared-array-buffer/#origin-trial)
+and wait until the new modes are available. We are not planning to terminate the
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:157]:
```suggestion
and waiting until the new modes are available. We are not planning to terminate the
```

**Review by mihajlija — APPROVED** [review]
Approved with comments, please resolve those before merging.

**Pull request merged**